src/App.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from 'react';
 import { getAllData } from './util/index';
-import LandingPage from "./components/LandingPage";
+import Home from "./components/Home";
```

Comment:
this file path needs to be updated to reflect the new file structure

Reply:
thank you, will fix